src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="search"
+    method="get"
+    data-qa="big"
+    class="form"
+  >
+      <label class="bigLabel">
+        <input
+        type="text"
+        class="field bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      </label>
+    </form>
+
+    <form
+    action="search"
+    method="get"
+    data-qa="small"
+    class="form"
+  >
```

Comment:
```suggestion
    <form
      action="search"
      method="get"
      data-qa="small"
      class="form"
    >
```
The same with indents.